prow/cmd/deck/main.go

```diff
@@ -157,19 +158,34 @@ var (
 			},
 			[]string{"path", "method", "status"},
 		),
+		httpRequestSize: prometheus.NewHistogramVec(
+			prometheus.HistogramOpts{
+				Name:    "deck_http_request_size_bytes",
+				Help:    "http request size in bytes for deck server",
+				Buckets: []float64{1024, 2048, 4096, 8192, 16384, 32768, 65536, 131072, 262144, 524288, 1048576, 2097152},
```

Comment:
We probably need a few buckets larger than 2MB. I just checked https://prow.k8s.io/prowjobs.js and it is currently 7.4MB zipped (83.4MB unzipped). 
Do you think using this many buckets will affect query performance significantly? If so we could increase the scale factor between the last few buckets.

Reply:
Wow, that's horrifying.

Reply:
Dropped the smaller buckets